apps/guide/src/content/04-popular-topics/05-collectors.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: Collectors
+category: Popular topics
+---
+
+# Collectors
+
+## Message collectors
+
+<p><DocsLink type="class" parent="Collector"><code>Collector</code>s</DocsLink> are useful to enable your bot to obtain *additional* input after the first command was sent. An example would be initiating a quiz, where the bot will "await" a correct response from somebody.</p>
+
+### Basic message collector
+
+For now, let's take the example that they have provided us:
```

Comment:
Who is "they"? @almostSouji can confirm the wording better than me, but I think we're trying to be more direct with our wording e.g., "Let's take a look at the following example" or something like that.

Reply:
It looks like it's been this way for the past 6 years, so I very much doubt it was intentional—just went unnoticed. Agree with your wording, I'd propose this exactly:

> Let's take a look at a basic message collector: